.babelrc.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
   presets: [
     [
-      'env',
+      '@babel/env',
```

Comment:
Is this really needed?

Reply:
yep that's because the new babel namespace

Reply:
yeah it's actually because we can't really know to default `env` to `babel-preset-env` or `@babel/preset-env` or `@x/preset-env` etc